contracts/ETHRegistrarController.sol

```diff
@@ -58,18 +64,24 @@ contract ETHRegistrarController is Ownable {
     }
 
     function makeCommitment(string memory name, address owner, bytes32 secret) pure public returns(bytes32) {
+        return makeCommitmentWithConfig(name, owner, secret, address(0), address(0));
```

Comment:
why do we set the resolver to 0, would this not be a good chance to set it to the default resolver already or is that a bad idea?

Reply:
This is for backwards-compatibility with users of the old registrar. We could default them to the public resolver, but it'd be a change in behaviour, and we'd have to embed the public resolver address in the controller contract, which I'd rather not do.